lib/option.gb

```diff
@@ -0,0 +1,29 @@
+class Option
+    def initialize(name=nil, value=nil)
```

Comment:
can we make the indentation to 2 spaces?

Reply:
Ah sorry!